### PR TITLE
Converted Config Utils to Kotlin

### LINKED ISSUE #3673: Convert ConfigUtils to Kotlin
**Summary:** 
As per the discussion from the [thread](https://github.com/commons-app/apps-android-commons/issues/747#issuecomment-591081603).

Convert ConfigUtils.java to Kotlin.

**Want to work on this issue?**
Yes

### PR DESCRIPTION
**Pull request by madhurgupta10**
**Description**
Fixes #3673 

**Tests performed**
Tested betaDebug on Google Pixel 2 with API level 29.

### CODE REVIEW

File: app/src/main/java/fr/free/nrw/commons/utils/ConfigUtils.kt [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import android.content.Context
+import android.content.pm.PackageManager
+import fr.free.nrw.commons.BuildConfig
+import java.util.*
+
+object ConfigUtils {
+    @JvmStatic
+    val isBetaFlavour: Boolean
+        get() = BuildConfig.FLAVOR == "beta"
```
Comment (macgills) [inline app/src/main/java/fr/free/nrw/commons/utils/ConfigUtils.kt:11]: this doesn't need to be a `get()` as it doesn't need to be evaluated every time someone accesses it

File: app/src/main/java/fr/free/nrw/commons/utils/ConfigUtils.kt [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            context.packageManager
+                .getPackageInfo(context.packageName, 0).versionName
+        } catch (e: PackageManager.NameNotFoundException) {
+            BuildConfig.VERSION_NAME
+        }
+    }
+
+    @JvmStatic
+    fun getVersionNameWithSha(context: Context): String {
+        return String.format(
```
Comment (macgills) [inline app/src/main/java/fr/free/nrw/commons/utils/ConfigUtils.kt:25]: Could be rewritten to use kotlin string interpolation

File: app/src/main/java/fr/free/nrw/commons/utils/ConfigUtils.kt [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+import fr.free.nrw.commons.BuildConfig
+import java.util.*
+
+object ConfigUtils {
+    @JvmStatic
+    val isBetaFlavour: Boolean
+        get() = BuildConfig.FLAVOR == "beta"
+
+    @JvmStatic
+    private fun getVersionName(context: Context): String {
```
Comment (macgills) [inline app/src/main/java/fr/free/nrw/commons/utils/ConfigUtils.kt:14]: Might be nicer as an extension function on context

File: app/src/main/java/fr/free/nrw/commons/utils/ConfigUtils.kt [hunk 4]
```diff
@@ -3,30 +3,22 @@ package fr.free.nrw.commons.utils
...
-        get() = BuildConfig.FLAVOR == "beta"
+    val isBetaFlavour: Boolean = BuildConfig.FLAVOR == "beta"
 
     @JvmStatic
-    private fun getVersionName(context: Context): String {
+    private fun Context.getVersionName(): String {
         return try {
-            context.packageManager
-                .getPackageInfo(context.packageName, 0).versionName
+            this.packageManager.getPackageInfo(this.packageName, 0).versionName
```
Comment (macgills) [inline app/src/main/java/fr/free/nrw/commons/utils/ConfigUtils.kt:14]: `this` can be omitted in both extension functions

Comment (madhurgupta10) [reply app/src/main/java/fr/free/nrw/commons/utils/ConfigUtils.kt:14]: Thanks, the extension functions are really cool.

Comment (macgills) [reply app/src/main/java/fr/free/nrw/commons/utils/ConfigUtils.kt:14]: Dope as hell, you can also have extension properties which is identical to a function but you can access it like

```
context.versionName
```

from

```
val Context.versionName
get() = //code
```

**Pull request merged**